checkout-service/database/seed/dataGenerator.js

```diff
@@ -0,0 +1,51 @@
+const Faker = require('faker');
+
+
+const CreateFakeData = () => {
+
+  // centsForPrice is biased to .99 because of its popularity in prices
+  let centsForPrice = [.49, .75, .85, .89, .99, .99];
+  let numberOfProducts = 100;
+  let maxInventory = 100;
+  let soldOutProductCount = 30
+  let soldOutProducts = [];
+
+  // Get an array of 30 arbitrary indexes that will be sold out
+  for (let j = 0; j < soldOutProductCount; j++) {
+    let randomProductIndex = Math.floor(Math.random() * Math.floor(maxInventory));
+    soldOutProducts.push(randomProductIndex);
+  }
+
+  let productsToSave = []
+
+  for (let i = 0; i < numberOfProducts; i++) {
+    let productId = i + 1;
+    let inventory;
+    // Creating an arbitrary index of centsForPrice to add realistic prices with cents to products
+    let centsIndex = Math.floor(Math.random() * Math.floor(centsForPrice.length));
+    // Get random number from faker round it and add the random cents from centsForPrice array. Prices from faker were too hight so I divided them by 10 to make them more like everyday item prices.
+    let price = Math.round(Number(Faker.commerce.price()) / 10)  + centsForPrice[centsIndex];
```

Comment:
If you want to avoid additional division, can you define max and min and get random number between those which Math.random allows and might be less code?

Reply:
That's right, well I guess I just wanted to us faker at least once. Lol